### PR TITLE
A couple of small fixes

### PR DESCRIPTION
**Pull request by phobos2077**
- changed <string> to <string.h> header in Game.cpp
- added exception when Texture could not be loaded

### CODE REVIEW

File: src/Game/Game.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 // C++ standard includes
 #include <sstream>
-#include <string>
+#include <string.h>
```
Comment (alexeevdv) [inline src/Game/Game.cpp]: Hm....why <string.h> ?

Comment (isage) [inline src/Game/Game.cpp]: For strdup.
As for <string> - it's already included in Game.h

Comment (alexeevdv) [inline src/Game/Game.cpp]: Do we really need strdup here?

Comment (isage) [inline src/Game/Game.cpp]: Yeah, that's the way putenv works.
>The string pointed to by string becomes part of the environment, so altering the string changes the environment.
But, i'm not sure we need this whole line.

Comment (alexeevdv) [inline src/Game/Game.cpp]: http://www.qnx.com/developers/docs/6.5.0_sp1/index.jsp?topic=%2Fcom.qnx.doc.neutrino_lib_ref%2Fp%2Fputenv.html
It is needed only when you are not using "literal string". But.... our string is hardcoded and it is quite "literal" :)

Comment (isage) [inline src/Game/Game.cpp]: Yeah, and will be deallocated when function ends ;)

Comment (alexeevdv) [inline src/Game/Game.cpp]: There was SDL_putenv function in SDL1.2 - can't find similar function in SDL2.0

Comment (isage) [inline src/Game/Game.cpp]: There should be SDL_setenv (which actually just calls setenv), but judging by setenv manpage it'll work without duplicating string. So yeah, we could use that (SDL_setenv version, as it deals with platform-specific env) and drop strdup.

**Pull request merged**